### PR TITLE
Navegg UserID Submodule: conform with pub storage configuration

### PR DESCRIPTION
**Pull request by p-ribeiro**
## Type of change
- [x] Bugfix
- [x] Other

## Description of change
Fixing the problems described at https://github.com/prebid/Prebid.js/issues/10710.
We are now saving the ID in cookie or html5, respecting the storage.type parameter. 
We also changed our logic to get the ID from our API.

### CODE REVIEW

**Comment by p-ribeiro** [comment]
Can someone give me an explanation on why did my test failed? It seems that what failed are another modules tests. 

For example the IntentQ and imuId module...

```

166) should return expired is true
     imuId module getLocalData()
     AssertionError: expected { Object (id, ppid, ...) } to deeply equal { Object (id, ppid, ...) }
    at Context.<anonymous> (/tmp/_karma_webpack_724924/commons.js:314947:49)

167) should save data in cookie if storage type is "cookie"
     IntentIQ tests
     AssertionError: expected null to not equal null
    at Context.<anonymous> (/tmp/_karma_webpack_724924/commons.js:317159:43)

168) "before each" hook for "should set cmpData.us_privacy if uspData exists"
     IntentIQ tests IntentIQ consent management within getId
```

File: modules/naveggIdSystem.js [hunk 1]
```diff
@@ -6,7 +6,7 @@
  */
 import { isStr, isPlainObject, logError } from '../src/utils.js';
 import { submodule } from '../src/hook.js';
-import { ajax } from '../src/ajax.js';
+import * as ajaxLib from '../src/ajax.js';
```
Comment (patmmccann) [inline modules/naveggIdSystem.js:9]: please only import what you need

**Review by patmmccann — CHANGES_REQUESTED** [review]
Thanks so much, one tweak requested

**Comment by p-ribeiro** [comment]
> Thanks so much, one tweak requested
The change was done as requested with my latest commit

**Comment by p-ribeiro** [comment]
May I ask a question here?

Is it permitted to add a cookie saver, as a default option, if the client does not implemente the storage options?

**Comment by patmmccann** [comment]
The parent module handles the storage writes

**Review by patmmccann — APPROVED** [review]

**Pull request merged**